### PR TITLE
[3825][16.0][IMP] stock_picking_accounting_date

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
[3825](https://www.quartile.co/web#id=3825&cids=3&menu_id=505&action=1457&model=project.task&view_type=form)

### CODE REVIEW

**Review by rinaldifirdaus — APPROVED** [review]
Work as expected

File: stock_picking_accounting_date/models/stock_picking.py [hunk 1]
```diff
@@ -18,7 +18,8 @@ def _compute_show_accounting_date(self):
             pick.show_accounting_date = False
             if pick.picking_type_code not in ("incoming", "outgoing"):
                 continue
-            if pick.move_ids.with_company(pick.company_id).product_id.filtered(
+            products = pick.move_ids.with_company(pick.company_id).product_id
+            if products.filtered(
                 lambda x: x.detailed_type == "product" and x.valuation == "real_time"
-            ):
+            ) and not products.filtered(lambda x: x.detailed_type == "consu"):
```
Comment (yostashiro) [inline stock_picking_accounting_date/models/stock_picking.py:24]: This change looks incorrect.  `show_accounting_date` should be true when the picking involves both stockable products and consumables.

**Review by yostashiro — CHANGES_REQUESTED** [review]
Please follow up on this comment as well, unless you are not convinced with the idea: https://github.com/qrtl/axls-custom/pull/53#discussion_r1360266723

**Comment by AungKoKoLin1997** [comment]
Closing this task. I misunderstood this comment https://github.com/qrtl/axls-custom/pull/53#discussion_r1359144500.

**Pull request closed without merging**